infra.bs

```diff
@@ -296,30 +296,30 @@ for working with them, in order to create common ground.
 <h3 id=lists>Lists</h3>
 
 <p>A <dfn export>list</dfn> is a specification type consisting of a finite ordered sequence of
-items.
+<dfn for=list lt=item export>items</dfn>.
```

Comment:
It appears we're not entirely consistent but I think this would be best as `for=list,stack,queue,set`. The usage sites appear a bit inconsistent too (e.g. sometimes in this spec we use `for=queue` for queues, sometimes we use `for=list`. But don't worry about that. I just think it'd be good to get the proper for="" values on the new exported terms in this PR.